decidim-stats/app/queries/decidim/stats/actions/comment.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Stats
+    module Actions
+      class Comment
+        def initialize(component, performers)
+          @component = component
+          @performers = performers
+        end
+
+        def query
+          @query ||=
+            component_resource_queries
```

Comment:
I don't fully understand why this method is suffixed "_queries".
Doesn't it returns a list of resource classes?

Reply:
It returns a list of `ActiveRecord::Relation` instead of the list of results, so they're queries that have not yet been performed